packages/typespec-ts/test/modularIntegration/azureCoreModel.spec.ts

```diff
@@ -0,0 +1,29 @@
+import { ModelClient } from "./generated/azure/core/model/src/index.js";
+import { assert } from "chai";
+
+describe("Traits Client", () => {
+  let client: ModelClient;
+
+  beforeEach(() => {
+    client = new ModelClient({
+      endpoint: "http://localhost:3002",
+      allowInsecureConnection: true
+    });
+  });
+
+  it("should get core model embeddingVector", async () => {
+    const result = await client.get();
+    assert.strictEqual(JSON.stringify(result), JSON.stringify([0, 1, 2, 3, 4]));
```

Comment:
why we need to stringify here? not sure the strictDeepEqual could work or not?

i don't like strinify because it earse the type information.

Reply:
Updated with `deepStrictEqual`